exercises/concept/lasagna/.docs/introduction.md

```diff
@@ -3,122 +3,120 @@
 The entire Julia track will require you to treat your solution like small libraries, i.e. you need to define functions, types etc. which will then be run against a test suite.
 For that reason, we will introduce named functions as the very first concept.
 
-## Functions
+Julia is a dynamic, strongly-typed programming langauge.
+The programming style is mainly functional, though with more flexibility than in languages such as Haskell.
 
-### Defining functions
+## Comments
 
-There are two common ways to define a named function in Julia:
+Two options are possible in Julia:
+- Single-line comments start with `#`
+- Multi-line comments start with `#=` and end with `=#`. Nesting is allowed.
 
-1. Using the `function` keyword
-
-    ```julia
-    function muladd(x, y, z)
-        return x * y + z
-    end
-    ```
+```julia
+# This is a single-line comment
 
-2. Using the "assignment form"
+x = 3  # This is an inline comment
 
-    ```julia
-    muladd(x, y, z) = x * y + z
-    ```
+#=
+	Multi-line comments can be used for longer explanations.
 
-    This is most commonly used for one-line function definitions or mathematical functions, where the function body is a single expression.
+	They are especially useful to comment out blocks of code during debugging.
+=#
+```
 
-### Invoking functions
+## Variables and assignment
 
-Invoking a function is done by specifying its name and passing arguments for each of the function's parameters:
+To create a variable, just assign a value to it:
 
-```julia
-# invoking a function
-muladd(10, 5, 1)
+```julia-repl
+julia> myvar = 42  # an integer
+42
 
-# and of course you can invoke a function within the body of another function:
-square_plus_one(x) = muladd(x, x, 1)
+julia> name = "Maria"  # strings are surrounded by double-quotes ""
+"Maria"
 ```
 
-## Integers and Arithmetic operations
+Types are an important subject in Julia, but for now it is best to ignore them them.
```

Comment:
Types aren't referenced before so this might be confusing to highlight this especially since there's no context for what a type is. Is it something to do with functions? Variables?

Reply:
Agree. This should be in `about.md`, not `introduction.md`

Reply:
Removed in the latest commit.